apisix/healthcheck_manager.lua

```diff
@@ -206,6 +210,15 @@ function _M.upstream_version(index, nodes_ver)
 end
 
 
+local function get_plugin_name(path)
+    -- Extract JSON path (after '#') or use full path
+    local json_path = path:match("#(.+)$") or path
+    -- Match plugin name in the JSON path segment
```

Comment:
For plugin names that contain hyphens, do we must to enclose them in quotation marks?